roles/openshift_hosted/defaults/main.yml

```diff
@@ -1,14 +1,34 @@
 ---
+##########
+# Common #
+##########
+openshift_hosted_infra_selector: "region=infra"
+r_openshift_hosted_use_calico_default: "{{ openshift_use_calico | default(False) }}"
+r_openshift_hosted_use_calico: "{{ r_openshift_hosted_use_calico_default }}"
+
+openshift_default_projects:
+  default:
+    default_node_selector: ''
+  logging:
```

Comment:
This can be left to openshift_logging to create.  @ewolinetz

Reply:
Possibly, again outside the scope of this PR, IMO.

Reply:
I agree, and also agree it probably is outside this PR.